server/database/seed.js

```diff
@@ -56,5 +74,5 @@ const seedManyUsers = (start, number) => {
     .catch(() => console.error('Batch unsuccessful'));
 };
 
-seedManyUsers(200, 10000);
+seedManyUsers(10206, 10);
```

Comment:
Why are you choosing to input 10206 and 10?

Reply:
oh, i was just playing with existing data and adding on to it 10 at a time after seeding with 10,000.